app/javascript/src/Policies/components/CustomPolicies.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import * as React from 'react'
+import {PolicyList} from 'Policies/components/PolicyList'
+import 'Policies/styles/policies.scss'
+import type {ShallowPolicy} from 'Policies/types/Policies'
+
+const CustomPolicies = ({policies = []}: {policies: Array<ShallowPolicy>}): React.Node => {
```

Comment:
I think we can avoid the return type, it's inferred.

Reply:
Also, and this is a personal preference / pattern that might seem over-verbose but it actually improves readability a lot. Since components are defined in their on files, it's nice to have the Props and State separately. This way whoever reads this component can easily find and understand its interface:
```js
type Props = {
  policies: Array<ShallowPolicy>
}

const CustomPolicies = ({ policies = [] }: Props) => {
```
You simply read a list instead of extracting pieces of information betwen braces, colons, commas, equals...

Reply:
it is when you have loads of them. When it's just one, it's easy to read it in the same line

Reply:
Didn't mean it was hard just not as nice as it could be.

Reply:
and that, again, is your opinion.